src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -88,6 +90,12 @@ export default class GroupComparisonStore {
         return this.appStore.isLoggedIn;
     }
 
+    @autobind
+    @action
+    public setWorkspaceGroups(map:ObservableMap<boolean>) {
+        this.isGroupInWorkspace = map.toJS();
```

Comment:
this sounds like a boolean but is actually a map.  why not a method that takes group as param and looks in map? also, why toJS?

Reply:
sure, I can split it into a map and a clearer getter method

toJS just makes a copy of it, I think it's a little cleaner that way than using the map thats passed in which the caller still has access to. and it's not necessary to be an observable map on this end, so toJS is good